src/main/menu.ts

```diff
@@ -176,6 +176,30 @@ export default class MenuBuilder {
         },
         enabled: false, // by default, gets updated when a project is entered
       },
+      {
```

Comment:
I think these playback actions should have their own separator so they appear separate on mac - otherwise they're grouped with the word correction actions which is a bit weird.

<img width="315" alt="Screen Shot 2022-09-24 at 11 06 48 am" src="https://user-images.githubusercontent.com/6735055/192073744-23dd5e05-977e-4695-b7b1-d16992adf68a.png">

Reply:
Even better would be if they had their own `Playback` menu, I don't really think it makes sense for them to be in the `edit` menu since they don't actually modify the project at all

Reply:
Okay no worries